routes/api/getshop.js

```diff
@@ -1,9 +1,15 @@
 const router = require("express").Router();
-const Shop = require("../../models/Shop");
+const Shop = require("../../controllers/shopController");
 
 // Shop routes
 router.route("/")
-  .get(medController.findAll)
-  .post(medController.create);
+  .get(Shop.findAll)

```

Comment:
Your code should work as is. I'm suggesting to rename your shop controller var to avoid confusion. 
`.get(shopController.findAll)`